tensorflow/contrib/lite/examples/ios/simple/RunModelViewController.mm

```diff
@@ -158,9 +166,8 @@ - (NSString*)runInferenceOnImage:(UIImage *)image {
     NSString* graph_path = [self filePathForResourceName:@"mobilenet_v1_1.0_224" withExtension: @"tflite"];
     
     std::unique_ptr<tflite::FlatBufferModel> model(tflite::FlatBufferModel::BuildFromFile([graph_path fileSystemRepresentation]));
-    if (!model) {
-        LOG(FATAL) << "Failed to mmap model " << graph;
-    }
+    NSAssert(model, @"Failed to mmap model %@", graph_path);
+
     LOG(INFO) << "Loaded model " << graph;
```

Comment:
So graph is not going to be anything valid here. Change to graph_path.UTF8String, or change LOG() over to NSLog?